test/table.ts

```diff
@@ -1698,6 +1698,39 @@ describe('BigQuery/Table', () => {
           })
           .emit('writing');
       });
+
+      describe('with BIGQUERY_EMULATOR_HOST baseUrl set', () => {
```

Comment:
I don't believe these tests should live here, as this is behavior that is performed in the `BigQuery` class.